kratos/python_scripts/python_linear_solver_factory.py

```diff
@@ -168,3 +168,28 @@ def ConstructSolver(configuration):
         KM.Logger.PrintInfo("Linear-Solver-Factory",\
             "Constructing a regular (non-complex) linear-solver")
         return KM.LinearSolverFactory().Create(configuration)
+
+def CreateDirectLinearSolver(settings):
+    if not settings.Has("linear_solver_settings"):
+        settings.AddEmptyValue("linear_solver_settings")
+    linear_solver_configuration = settings["linear_solver_settings"]
+    if linear_solver_configuration.Has("solver_type"): # user specified a linear solver
+        return ConstructSolver(linear_solver_configuration)
+    else:
```

Comment:
this method should only do the things after `else`
Otherwise it could also create an iterative solver

Reply:
True (I think that the only limitation is the linear solver, the iterative would be fine I think)